### PR TITLE
add fit finder tool

### LINKED ISSUE #8: Bra Fit Finder
- [ ] Import and implement https://www.maidenform.com/bra-finder

### PR DESCRIPTION
**Pull request by hannessolo**
Fix #8 

Test URLs:
- Before: https://main--maidenform--hlxsites.hlx.page/bra-finder
- After: https://bra-finder--maidenform--hlxsites.hlx.page/bra-finder
- Before: https://main--maidenform--hlxsites.hlx.page/bra-fit-calculator/results?band-size=32&cup-size=34
- After: https://bra-finder--maidenform--hlxsites.hlx.page/bra-fit-calculator/results?band-size=32&cup-size=34
- Before: https://main--maidenform--hlxsites.hlx.page/bra-fit-calculator/results
- After: https://bra-finder--maidenform--hlxsites.hlx.page/bra-fit-calculator/results

### CODE REVIEW

File: blocks/bra-finder/bra-finder.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  setSelectedTab(tab, i) {
+    this.deselectAllTabs();
+    tab.setAttribute('aria-selected', 'true');
+    this.rootUl.parentElement.querySelectorAll('[role="tabpanel"]')[i].removeAttribute('hidden');
+  }
+}
+
+class SizeSelector {
+  constructor() {
+    this.bandSizes = [32, 34, 36, 38, 40, 42];
```
Comment (herzog31) [inline blocks/bra-finder/bra-finder.js:42]: Could you read those from the JSON as well?

File: blocks/bra-finder-results/bra-finder-results.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+export default async function decorate(block) {
```
Comment (herzog31) [inline blocks/bra-finder-results/bra-finder-results.js:1]: The actual page also has a `< back to bra fit guide` button. Could you please add this as well.

**Review by herzog31 — APPROVED** [review]
LGTM, just minor comments. Let's also update the fonts once we have those clearly defined as CSS variables.

**Comment by hannessolo** [comment]
@herzog31 I will merge this and take care of those changes in the next PR, including the font updates

**Pull request merged**